tests/test_done_state.py

```
@@ -0,0 +1,46 @@
+def test_apply_done_decision_requires_confirmation():
+    finding = {"id": "f-1", "status": "in_progress"}
+    decision = DoneDecision(
+        state="resolved",
+        rationale="Fixed by PR #12",
+        actor="rico",
+        confirmation=False,
+    )
+
+    try:
+        apply_done_decision(finding, decision, now_iso="2026-05-12T19:10:00+00:00")
+        assert False, "expected confirmation_required"
```

Ruff (0.15.12)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)